printlabels/printlabels.js

```diff
@@ -45,7 +42,7 @@ let app = undefined;
 let data = {
   status: 'waiting',
   labels: null,
-  template: startTemplate,
+  template: null,
```

Comment:
Is the order of calls of `onOptions` and `onRecords` guaranteed, so that template gets set by the time it's needed?

Reply:
You are right, there should be a template from the start, as this order is a side effect of fetching records in the onRecords handler.